fix(curriculum): add `target="_blank"` to link in challenge (#41870)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/nest-an-anchor-element-within-a-paragraph.md

`target` is an anchor tag attribute that specifies where to open the link. The value `_blank` specifies to open the link in a new tab. The `href` is an anchor tag attribute that contains the URL address of the link:  

```html

> <a href="http://freecodecamp.org" target="_blank"> ... </a>

Reviewer: This is probably out-of-scope, but while we are at it, could you help update the URL as well? It's for consistency, and I personally feel a bit safer when I see a link with `https` protocol 😄 

```suggestion
<a href="https://www.freecodecamp.org" target="_blank"> ... </a>
```

```

The text, `link to freecodecamp.org`, within the `a` element is called <dfn>anchor text</dfn>, and will display the link to click:

```html

> <a href=" ... " target="_blank">link to freecodecamp.org</a>

Reviewer 2: ```suggestion
<a href=" ... " target=" ... ">link to freecodecamp.org</a>
```

Yep I think I'm in favour of this syntax.

Reviewer 3: Nicholas, I cannot find a reference to this syntax anywhere? Is it really worth it to save 1 character?

Reviewer 2: I'm fine with it either way - it mirrors the `href` syntax right next to it.

If we want to leave the text there, we should put the `href` text there too.

Reviewer 3: 🤦 My bad... I did not realise this was in the example - I thought this was the _fix_

```

The final output of the example will look like this:  

> Here's a <a href="http://freecodecamp.org" target="_blank">link to freecodecamp.org</a> for you to follow.

Reviewer: Here as well

```suggestion
Here's a <a href="https:/www./freecodecamp.org" target="_blank">link to freecodecamp.org</a> for you to follow.
```

Reviewer: @nhcarrigan Do you think we need `rel="noopener noreferrer"` too?

Reviewer 2: You have a typo in the suggestion here 😉 

I think those aren't needed from a *security* standpoint here, because we're linking to ourselves. However, I do agree that covering them wouldn't hurt.

That being said, I think that's outside of the scope of this PR - we should create an issue to update the examples and instruction text, possibly.

Reviewer: Oops 😝 

Yeah I agree that this shouldn't block the PR (that's why I didn't request the change). I also think it's easier to maintain the consistency if we audit and make the update in the same PR, rather than randomly.

I'll let you sign off on the PR then 🙂
